common/list/lists-en.html

```diff
@@ -0,0 +1,583 @@
+---
+{
+	"title": "GCWeb Lists example",
+	"language": "en",
+	"altLangPage": "lists-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-03-05"
+}
+---
+
+<h1> Lists</h1>
+
+<h4> Unordered lists</h4>
+<h5>Appearance</h5>
```

Comment:
This don't match the heading sequence of the page. (see the following WCAG technique: [H42](https://www.w3.org/WAI/WCAG22/Techniques/html/H42), [G141](https://www.w3.org/WAI/WCAG22/Techniques/general/G141), [F43](https://www.w3.org/WAI/WCAG22/Techniques/failures/F43)